jekyll/_cci2/workflows.md

```diff
@@ -49,7 +66,7 @@ workflows:
 
 ## Sequential Job Execution Example
 
-The following example shows a workflow with four sequential jobs. The jobs run according to configured  requirements, each job waiting to start until the required job finishes successfully as illustrated in the diagram. 
+The following example shows a workflow with four sequential jobs. The jobs run according to configured requirements, each job waiting to start until the required job finishes successfully as illustrated in the diagram. 
```

Comment:
sorry, this content didn't change in the latest revision, so I will leave this for now